src/panels/config/server_control/ha-config-server-control.ts

```diff
@@ -41,6 +41,14 @@ const reloadableDomains = [
   "command_line",
   "filter",
   "statistics",
+  "generic",
```

Comment:
```suggestion
  "generic_camera",
```

Reply:
Sadly, it actually has the domain `generic`

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/generic/camera.py